<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderEdit</class>
 <widget class="QMainWindow" name="orderEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderScheduler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createNewIterationBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>创建新的迭代</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewHisIterationBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>查看历史迭代</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>仅保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAndArrangeBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>保存并排期</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>262</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>当前迭代：</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit"/>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>——</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2"/>
   </widget>
   <widget class="QPushButton" name="createNewOrderBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>创建工单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createNewTestOrderBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>快速创建测试任务工单</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action2">
   <property name="text">
    <string>创建新的迭代</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>查看历史迭代</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
